### DISCUSSION TITLE
Can you get banned when using this?

### DISCUSSION THREAD

**Discussion by Stickpage2020** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#3] [General]
I hope this is safe

**Comment by WakuWakuPadoru** [comment]
Hi. As stated, this doesn't change the game files in any way. 

Whenever you make changes to the graphics settings, the settings are saved to a file and that's how the game remembers your settings. Instead of changing the game directly, this unlocker only changes your settings.

So yup! Should be safe!
